src/test/unit_tests/test_namespace_fs.js

```diff
@@ -798,6 +807,10 @@ mocha.describe('namespace_fs folders tests', function() {
             }, dummy_object_sdk, read_res);
             assert.equal(read_res.buffers.length, 1);
             assert.equal(read_res.total_length, 100);
+            let get_md_res = await ns_tmp.read_object_md({
```

Comment:
I think I am missing something, why are we issuing `read_object_md` call twice?

Reply:
correct, removed